mitodl/open-discussions
# Login popup for anonymous user vote buttons on post detail page

#### Pre-Flight checklist


#### What are the relevant tickets?
Partially addresses #1051

#### What's this PR do?
Displays a popup with login and signup buttons when an anonymous user clicks vote buttons for a post or comment on a post detail page.

#### How should this be manually tested?
Go to the detail page for a post in a public channel, as an anonymous user.  Click the vote buttons for the post and for some of the comments.  You should see a popup with login and signup buttons that should bring you to the appropriate page.  Clicking the upper-right x in the popup or anywhere outside the popup should close it. 

#### Screenshots
<img width="327" alt="screen shot 2018-08-20 at 4 02 29 pm" src="https://user-images.githubusercontent.com/187676/44363553-7b8fc400-a492-11e8-9f8a-cc24034b0466.png">
